.github/workflows/CI.yml

```diff
@@ -16,60 +16,34 @@ jobs:
       - uses: actions/checkout@v4
       - name: Spotless Check
         run: ./gradlew spotlessCheck
-  javadoc:
-    if: github.repository == 'opensearch-project/opensearch-ai-flow-framework'
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Javadoc CheckStyle
```

Comment:
Will we have reviews on this feature branch? I see you removed some checks here

Reply:
These checks are not removed but handled in the `./gradlew check` suite already as mentioned in the description since this a temporary feature branch.

Reply:
Awesome, didn't realize check does all of this as well, used to build in the plugin world :)

Reply:
Let's also properly handle getting this into main with cherry-pick PRs to preserve the original contributor info.